src/resources/views/crud/columns/range.blade.php

```diff
@@ -0,0 +1,26 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['text'] = $column['default'] ?? '-';
+    $max_value = (isset($column['max'])) ? $column['max'] : '100';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = ($column['value'] / $max_value) * 100;
+    }
+@endphp
+
+@php
+    if($column['text'] != "-"):
+@endphp
```

Comment:
Why use `@php` when we have `@if`? Would improve readability and solve the indentation problem here:

```suggestion
@if ($column['text'] != "-"):
```